Client/Frontend/TabBar/SearchViewController.swift

```diff
@@ -52,8 +52,15 @@ class SearchViewController: UIViewController {
 
 extension SearchViewController: UITableViewDataSource {
     func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+        var searchEngine: OpenSearchEngine?
+        if indexPath.item == 0 {
```

Comment:
Maybe it is nicer if all this is done just once before the tableview is shown? Now you loop through the list for every cell. I don't think that is noticable in performance but from a code point of view it is nicer to build the list table contents once at view controller setup.

Also, my documentation says that `NSIndexPath.item` is for the `UICollectionView`, and that it is not available in Swift? I think you want to use `NSIndexPath.row` instead.